models/attachment/src/index.ts

```diff
@@ -80,8 +82,19 @@ export class TSavedAttachments extends TPreference implements SavedAttachments {
   declare attachedTo: Ref<Attachment>
 }
 
+@Model(attachment.class.Drawing, core.class.Doc, DOMAIN_ATTACHMENT)
+export class TDrawing extends TDoc implements Drawing {
```

Comment:
I think that eventually we will need separate plugin for drawings.

Reply:
Yes, probably, when it grows. But semantically they are "attachments" because they don't make sence without a parent, so I decided to put them in the "attachment" plugin for now.